### PR TITLE
runtime: remove `When serialized in JSON, the format MUST adhere to the following pattern`

### LINKED ISSUE #1177: runtime.md: State MUST be serialized with specific indentation pattern?
> When serialized in JSON, the format MUST adhere to the following pattern:
> 
> ```json
> {
>     "ociVersion": "0.2.0",
>     "id": "oci-container1",
>     "status": "running",
>     "pid": 4422,
>     "bundle": "/containers/redis",
>     "annotations": {
>         "myKey": "myValue"
>     }
> }
> ```

https://github.com/opencontainers/runtime-spec/blob/v1.0.2/runtime.md#state

Does the above sentence demands that a state JSON "MUST" adhere to the specific indentation pattern?
I assume not, but what does "the following pattern" mean then?

### PR DESCRIPTION
**Pull request by AkihiroSuda**
The sentence looked like as if it required a specific indentation pattern.

Fix #1177

### CODE REVIEW

**Review by tianon — APPROVED** [review]

**Review by giuseppe — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

File: runtime.md [hunk 1]
```diff
@@ -32,7 +32,9 @@ The state of a container includes the following properties:
 
 The state MAY include additional properties.
 
-When serialized in JSON, the format MUST adhere to the following pattern:
```
[the remark this answers is not in the document]
Comment (thaJeztah) [reply runtime.md:35]: (👋) You mean the fields included? Aren't those already described in the runtimeState section above? Or do you mean the formatting of the JSON?

[the remark this answers is not in the document]
Comment (duglin) [reply runtime.md:35]: HOWDY! :-)

I mean the format of the JSON. For example, while it seems obvious what it should look like, technically w/o the MUST this JSON could be generated and claim to be compliant:
```
{
    "metadata" {
      "ociVersion": "0.2.0",
      "id": "oci-container1",
      "annotations": {
          "myKey": "myValue"
      }
    },
    "bundle": "/containers/redis",
    "state": {
      "status": "running",
      "pid": 4422
    }
}
```
since it still has all of the specified properties. The problem doesn't really show itself in JSON as well as other formats like XML where values could appear as new XML elements or as attributes on XML elements, but I believe the problem still does exist.

Basically, w/o the MUST we're doing "spec by example".  

If there is an issue here (which I don't really think there is), we should resolve it by just adding a statement that whitespace isn't significant (per the json spec: https://www.rfc-editor.org/rfc/rfc7159  and https://www.json.org/json-en.html )

[the remark this answers is not in the document]
Comment (AkihiroSuda) [reply runtime.md:35]: I believe your concern is already covered by https://github.com/opencontainers/runtime-spec/blob/main/schema/state-schema.json

[the remark this answers is not in the document]
Comment (duglin) [reply runtime.md:35]: If the spec said that people MUST adhere to the json schema then probably yes, but it doesn't say that. W/o that normative link there's no requirement to even look at it.

[the remark this answers is not in the document]
Comment (duglin) [reply runtime.md:35]: btw, there's a reason the word "pattern" was used instead of just saying "it MUST adhere to the following".

[the remark this answers is not in the document]
Comment (AkihiroSuda) [reply runtime.md:35]: Updated the PR:
```markdown
When serialized in JSON, the format MUST adhere to the JSON Schema [`schema/state-schema.json`](schema/state-schema.json).
```

**Review by thaJeztah — APPROVED** [review]
LGTM if this helps removing the ambiguity 👍

**Comment by duglin** [comment]
LGTM
